test/controllers/util/SubmissionDisplayHelperSpec.scala

```diff
@@ -155,16 +167,28 @@ object SubmissionDisplayHelperSpec {
 
   val uuid: String = UUID.randomUUID().toString
   val uuid_2: String = UUID.randomUUID().toString
+  val uuid_3: String = UUID.randomUUID().toString
   val eori: String = "GB167676"
   val ducr: String = createRandomAlphanumericString(16)
   val lrn: String = createRandomAlphanumericString(22)
   val mrn: String = "MRN87878797"
   val mrn_2: String = "MRN12341234"
+  val mrn_3: String = "MRN12341235"
   val conversationId: String = "b1c09f1b-7c94-4e90-b754-7c5c71c44e11"
   val conversationId_2: String = "b1c09f1b-7c94-4e90-b754-7c5c71c55e22"
+  val conversationId_3: String = "b1c09f1b-7c94-4e90-b754-7c5c71c55e22"
```

Comment:
Just checking...   conversationId_3 is the same as conversationId_2 (and _3 appears to be unused)